### PR TITLE
Updated advanced configuration for app framework page for v3

### PR DESCRIPTION
**Pull request by 1000TurquoisePogs**
The page "[Configuring Zowe Application Framework](https://docs.zowe.org/stable/user-guide/mvd-configuration)" (mvd-configuration.md) is currently accurate for v3, but just overall dated.

There are things in here that don't need to be said for v3, or can be said more simply because documentation elsewhere has been created and refined (AT-TLS documentation, MFA, how to access)

There are things here (v1 environment variable compatibility and HTTP mode) that I do not want documented because there is no reason why anyone should do that these days.

And finally, the name in the sidebar and the similar APIML page do not match a naming scheme.
image

I have updated this page's title to match APIML's so that there is some consistency.

### CODE REVIEW

File: docs/user-guide/mvd-configuration.md [hunk 1]
```diff
@@ -38,16 +20,12 @@ If the mediation layer is not used, ZSS directly at `https://<zowe.externalDomai
 
 ### app-server configuration
 
-The app-server uses the Zowe server configuration file for customizing server behavior. For a full list of parameters, requirements, and descriptions, see [the json-schema document for the app-server](https://github.com/zowe/zlux/blob/v2.x/staging/schemas/zlux-config-schema.json) which describes attributes that can be specified within the configuration file section `components.app-server`
```
Comment (janan07) [inline docs/user-guide/mvd-configuration.md:41]: Multiple links are not found with these changes. Please see the broken links report.

**Review by janan07 — CHANGES_REQUESTED** [review]
Please see the broken links report for broken links introduced in this PR

**Comment by 1000TurquoisePogs** [comment]
I think this PR's content could be ready to go, but I'm just awaiting review from more webui members to see if they have other feedback.

**Review by janan07 — APPROVED** [review]

**Comment by janan07** [comment]
The link checker test now passes.

**Comment by janan07** [comment]
Broken links have been fixed

**Pull request merged**